src/geo/lng_lat.js

```diff
@@ -42,7 +42,11 @@ class LngLat {
      * wrapped.lng; // = -73.9749
      */
     wrap() {
-        return new LngLat(wrap(this.lng, -180, 180), this.lat);
+        if (this.lng <= -180 || this.lng > 180) {
```

Comment:
Any particular reason to change this behavior? We'd want to at least update the API documentation above, which says that `wrap` always returns a new object.